pom.xml

```diff
@@ -229,6 +233,12 @@
                 <artifactId>testcontainers</artifactId>
                 <version>${org.testcontainers.version}</version>
             </dependency>
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
+                <artifactId>auth-crt</artifactId>
+                <version>${aws.java.sdk.version}</version>
+                <scope>runtime</scope>
```

Comment:
IMHO `runtime` is correct (not `compile`) @maschnetwork ?

Reply:
I haven't found any documentation - I assume runtime is okay but we need to verify this.

Reply:
Well, it's more a Java thing here. I tested it and it seems to work as I can see the generated Uber JAR containing `auth-crt`.

Reply:
Yeah I mean I haven't found any examples or documentation on the auth-crt module. If tests were passing - then it should be fine.

Reply:
Same :/